src/graphql/resolvers/ssb.js

```diff
@@ -237,6 +241,24 @@ module.exports = function Resolvers (ssb) {
       }
     },
 
+    Feed: {
+      threads: ({ threads, opts }) => {
+        const { limit = 10, after } = opts
+        const startIndex = after
+          ? threads.findIndex(({ messages }) => {
+            const key = messages[0].key
+
+            return after === key
+          }) + 1
+          : 0
+
+        return threads.slice(startIndex, startIndex + limit)
+      },
+      threadsCount: ({ threads }) => threads.length
```

Comment:
where in the design do we need threadsCount? 
There will be a much cheaper way of calculating this I think?

Reply:
I think if this is needed write a custom query for it. I can help if needed